chapter_appendix-tools-for-deep-learning/aws.md

```diff
@@ -217,14 +217,14 @@ After creating the `d2l` environment, activate it and install `pip`.
 
 ```bash
 conda activate d2l
-conda install python=3.7 pip -y
+conda install python=3.7 pip notebook -y
```

Comment:
no need for users, jupter is in https://github.com/d2l-ai/d2l-book/blob/master/setup.py#L5